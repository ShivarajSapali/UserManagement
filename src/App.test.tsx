import React from 'react'
import { App } from './App'
import renderer from 'react-test-renderer'
import { Provider } from 'react-redux'
import { getStore } from './store';

const store = getStore();

it('App snapshot test', () => {
    const component = renderer.create(<Provider store={store}><App /></Provider>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})