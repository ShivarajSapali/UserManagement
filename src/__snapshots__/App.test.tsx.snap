// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<main>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined add-user-btn MuiButton-outlinedPrimary"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Add User
    </span>
  </button>
  <div />
  <div
    className="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <table
      aria-label="customized table"
      className="MuiTable-root makeStyles-table-1"
    >
      <thead
        className="MuiTableHead-root"
      >
        <tr
          className="MuiTableRow-root MuiTableRow-head"
        >
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-86"
            scope="col"
          >
            <div>
              First Name
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-86"
            scope="col"
          >
            <div>
              Last Name
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-86"
            scope="col"
          >
            <div>
              Email
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-86"
            scope="col"
          >
            <div>
              DOB
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-86"
            scope="col"
          >
            <div>
              Gender
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-86"
            scope="col"
          >
            <div>
              Is Admin
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-86"
            scope="col"
          >
            <div>
              Business Unit
            </div>
          </th>
          <th
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-head-86"
            scope="col"
          >
            <div>
              Actions
            </div>
          </th>
        </tr>
      </thead>
      <tbody
        className="MuiTableBody-root"
      >
        <tr
          className="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-101"
        >
          <td
            aria-sort={null}
            className="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-body-87"
          >
            No data loaded
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</main>
`;
